cmd/flux/install.go

```diff
@@ -119,6 +119,20 @@ func installCmdRun(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	kubeClient, err := utils.KubeClient(kubeconfigArgs, kubeclientOptions)
```

Comment:
This happens way to early and it breaks `--export` which shouldn't require a cluster being available. Please move this right before `utils.Apply`.